ql/pricingengines/swaption/blackswaptionengine.hpp

```diff
@@ -273,6 +273,9 @@ namespace QuantLib {
                                             swap.floatingSchedule().dates().back());
         results_.additionalResults["swapLength"] = swapLength;
```

Comment:
I would probably move this after `swapLength` is modified?

Reply:
Yesterday I thought no, but today I agree it's better to be consistent :-)